source/FreeRTOS_Sockets.c

```diff
@@ -712,7 +712,8 @@ Socket_t FreeRTOS_socket( BaseType_t xDomain,
  *                   If the value is 'portMAX_DELAY' then the function will wait
  *                   indefinitely for an event to occur.
  *
- * @return The socket which might have triggered the event bit.
+ * @return The event bits (event flags) value for the socket in which an event occurred.
```

Comment:
```suggestion
 * @return The event bits (event flags) value for the socket set in which an
 *         event occurred. If any socket is signaled (using
 *         FreeRTOS_SignalSocket), then eSELECT_INTR is returned.
```

Reply:
Isn't `signaled` written with two l's?

Beside that, @tony-josi-aws, are using `ipconfigSUPPORT_SIGNALS`? Does it work well? Do you use it ti interrupt a call to `FreeRTOS_select()`?

Reply:
@htibosch fixed the typos. 

Havent used `ipconfigSUPPORT_SIGNALS` yet. Will update once I use it.

Reply:
@htibosch this PR is for a forum post here: https://forums.freertos.org/t/freertos-plus-tcp-freertos-select-incorrect-return-value-documentation/16333.
The post creator must be using select functions - I think they can provide you good feedback.